explore/src/main/scala/explore/tabs/AsterismEditorTile.scala

```diff
@@ -59,6 +59,7 @@ object AsterismEditorTile:
     backButton:        Option[VdomNode] = none
   )(using FetchClient[IO, ObservationDB], Logger[IO]): Tile = {
     // Save the time here. this works for the obs and target tabs
+    // It's OK to save the viz time for executed observations, I think.
     val vizTimeView =
       vizTime.withOnMod(t => ObsQueries.updateVisualizationTime[IO](obsIds.toList, t).runAsync)
```

Comment:
This needs an update I think

Reply:
Actually, it looks like it is still called `ObsQueries.updateVisualizationTime` in your PR. 😄 So, I don't think this needs to change. Unless there is another reason.